docs/redirects.js

```diff
@@ -232,7 +232,12 @@ const CURRENT = [
     destination: '/docs/graphql/overview',
     permanent: false,
   },
-
+  /* Telemetry */
+  {
+    source: '/telemetry',
+    destination: '/docs/reference/telemetry',
+    permanent: true,
+  },
```

Comment:
We don't need this. There is no `/telemetry` URL now.

Reply:
The logged notification in the CLI when running `keystone dev` points users to `/telemetry` to make is easier for a user to type in a URL it say:
`For more details visit: https://keystonejs.com/telemetry`